### DISCUSSION TITLE
Is there available pagination in Carousel element, So that the information of the next slide is loaded only after clicking?

### DISCUSSION THREAD

**Discussion by Dozez13** [quasarframework/quasar#9529] [General - Components / Directives / etc]
if it is so how can I use it?

**Comment by ontwikkelfabriek** [comment]
@Dozez13 

The carousel does load the next slide after you `click` or swipe to the next slide.

See here :
https://quasar.dev/vue-components/carousel#example--image-slides

Take a look at you network tab in chrome inspector and you'll see that the next image is requested/loaded only after you change slides.
